doc/install/installers.rst

```diff
@@ -0,0 +1,60 @@
+.. include:: ../links.inc
+
+MNE-Python installers
+=====================
+
+MNE-Python installers are the easiest way to install MNE-Python and
+all dependencies. They also provide many additional
+Python packages and tools, including the `Spyder`_ development environment.
+Got any questions? Let us know on the `MNE Forum`_!
+
+.. raw:: html
+    :file: installers_platform_selector.html
+
+First steps
+^^^^^^^^^^^
+
+The installer adds menu entries on Linux and Windows, and several application
+bundles to the ``Applications`` folder on macOS.
+
+.. rst-class:: list-unstyled
+.. rst-class:: mx-5
+.. rst-class:: mt-4
+.. rst-class:: mb-5
+
+- |code| |ensp| Use **MNE Spyder** to start writing your own analysis scripts right away, or to run one of our examples from this website.
+
+  .. rst-class:: mt-3
+- |desktop| |ensp| With **MNE System Info**, list the versions of all installed MNE-Python-related packages.
+
+  .. rst-class:: mt-3
+- |terminal| |ensp| The **MNE Prompt** drops you into a command line interface with a properly activated MNE-Python environment.
+
+
+.. note::
+   ⏳ Depending on your system, it may take a little while for these
+   applications to start, especially on the very first run – which may take
+   particularly long on Apple Silicon-based computers. Subsequent runs should
+   usually be much faster.
+
+
+VS Code Setup
+^^^^^^^^^^^^^
+
+If you want to use MNE-Python with Visual Studio Code, you need to tell the
+VS Code Python extension where to find the respective Python executable. To do
+so, simply start the **MNE Prompt**. It will display several lines of
+information, including a line that will read something like:
```

Comment:
this more I think about this the more I think it's too much information at this stage. Also this is not specific to the installers. Note that `Visual Studio Code`_ is the standard ref in our doc and it already explained elsewhere.

Reply:
Note that this **is** specific to the installers. By default, the Python environments installed via our installers won't show up in the VS Code Python interpreter picker, so users need a simple way to figure out how to locate the Python binary so they can tell VS Code about its path.